### ISSUE TITLE
some parameters like capabilities won't show up in converted output

### ISSUE THREAD

**Issue by surajssd** [errordeveloper/kubegen#18]
```bash
✔ ~/go/src/github.com/errordeveloper/kubegen/examples/modules/sockshop [master|…18] 
15:12 $ cat cart.yml | grep -A 6 capabilities
      capabilities:
        drop:
          - all
        add:
          - NET_BIND_SERVICE
      readOnlyRootFilesystem: true
    volumeMounts:
--
      capabilities:
        drop:
          - all
        add:
          - CHOWN
          - SETGID
          - SETUID
```

when I do the conversion:

```bash
✔ ~/go/src/github.com/errordeveloper/kubegen/examples/sockshop-prod.d [master|…18] 
15:11 $ cat cart.yaml | grep -A 6 capabilities   
```

and in test also none found:

```bash
✔ ~/go/src/github.com/errordeveloper/kubegen/examples/sockshop-test.d [master|…18] 
15:14 $ cat cart.yaml | grep -A 6 capabilities
```

kubegen I have downloaded the latest version yesterday.

**Comment by errordeveloper** [comment]
@surajssd thanks very much for reporting it! You are right, this appears to be unimplemented right now, but it should be easy to fix.

**Comment by errordeveloper** [comment]
@surajssd see the fix in  ff2accc. I am not very happy with either of the two copiers I've ended-up using, but this other copier fixes the bug... So `v1.PodSecurityContext` is easy, and go1.8 happily converts my struct to the regular one, but conversion breaks with `v1.SecurityContext` as `Capabilities` type has to be defined in order to customise the filed tags for HCL. Once the project grows up, I'd like to find a better way to do these things. One option would be to drop HCL support, but for now, there is sufficient interest in HCL.
